Application/Frontend/components/Server/Member_List.tsx

```diff
@@ -51,6 +62,90 @@ export function MemberList({isAdmin}: any) {
       }
   }
 
+  const fetchUserName = async (memberIDs: String[]) => {
+    try {
+      const response = await fetch('/api/ID-to-User-Name', {
```

Comment:
Good API name, but `ID-to-username` sounds better. No need to change this though as that is just a preference.